pages/form/WorkArrangement.jsx

```diff
@@ -0,0 +1,186 @@
+import { useState } from "react";
+import { 
+  Box,
+  Heading,
+  Text,
+  Radio,
+  RadioGroup,
+  Collapse,
+  Flex
+} from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+import useForm from "../../components/FormProvider";
+import Q1Progress from "../../public/images/progress-bar/q1-progress-dots.svg";
+import Q3Cloud from "../../public/images/clouds/cloud-q3.svg";
+import LinkButton, { BackButton } from "../../components/LinkButton/LinkButton";
+import { useRouter } from "next/router";
+import { sendLogs } from "../../utils/sendLogs";
+
+const workFromHome = "I work fully from home";
+const workOnSite = "I work fully on-site";
+const hybridMode = "I spend some days working on-site"
+
+export default function WorkArrangement() {
+
+  // code from previoud version below, no changes made
+
+  const { answers, setAnswers } = useForm();
+  const [ workMode, setWorkMode ] = useState(answers.workMode);
+
+  const saveAnswers = () => {
+    // saving radio button selection
+    setAnswers(prev => ({ ...prev, workMode: workMode }));
+  }
+
+  const router = useRouter();
+
+  const logMessage = (msg) => {
+    let incentiveMsg = () => {
+      if (!!answers.incentive) {return "<filled>"}
+      else return "<empty>"
+    }
+    return {
+      page: router.pathname,
+      event: msg,
+      ...answers,
+      workMode: workMode,
+      incentive: incentiveMsg(),
+    }
+  }
+
+  // function to save data and show logs on save
+  const saveDataAndShowLog = (logMsg) => {
+    saveAnswers();
+    sendLogs(logMessage(logMsg));
+  };
+
+  return (
+    <Layout 
+      isText={true} 
+      Progress={Q1Progress}
+    >
+      <Box 
+        pos="absolute" 
+        top={["2", "5"]} 
+        left={["2", "10"]}
+      >
+        <BackButton
+          href="/"
+          onClick={() => {
+            saveDataAndShowLog("Back button clicked");
+          }}
+        />
+      </Box>
+      <Q3Cloud />
```

Comment:
we probably need to change this cloud image to match the latest figma designs.
Might be easier to tackle that once all survey pages have been updated.
Have added the task of updating cloud pics to the [TODO list](https://trello.com/c/Nuhvxjes).

Reply:
Let's update them once the pages are ready. Also the progress bar, it's now different.